devicetypes/fibargroup/fibaro-co-sensor-zw5.src/fibaro-co-sensor-zw5.groovy

```diff
@@ -14,9 +14,9 @@ metadata {
 
 		attribute "coLevel", "number"
 
-		fingerprint mfr: "010F", prod: "1201", model: "1000"
-		fingerprint mfr: "010F", prod: "1201", model: "1001"
-		fingerprint mfr: "010F", prod: "1201"
+		fingerprint mfr: "010F", prod: "1201", model: "1000", deviceJoinName: "Fibaro Smoke Detector"
+		fingerprint mfr: "010F", prod: "1201", model: "1001", deviceJoinName: "Fibaro Smoke Detector"
+		fingerprint mfr: "010F", prod: "1201", deviceJoinName: "Fibaro Smoke Detector"

```

Comment:
Question here: Since CO detectors are distinct from Smoke Detectors, do we want to change the name here and on similar devices?